src/ALaDyn.F90

```diff
@@ -65,7 +65,7 @@ program ALaDyn
   call BUNCH_cycle
  end select
 
- call timing
+ !call timing
```

Comment:
Same as before, why remove timing?

Reply:
With the redefinition of dt so that dt*n_iter = tmax, the last time step which was computed to get to the exact final time (e.g. ALaDyn.F90 - line 120), is always avoided. So, we call timing just before calling data_out and then, since we are not making any other time steps, calling timing again outside the cycle may be redundant. Anyway, I'll restore it if you think it can be an important final check on the time when the main cycle has finished.

Reply:
ok thanks for the explanation